automatminer/automl/adaptors.py

```diff
@@ -18,6 +18,8 @@
     regression_or_classification
 from automatminer.base import AutoMLAdaptor, LoggableMixin
 
+from automatminer.automl.keras_wrapper import NnWrapper
```

Comment:
remove the space between imports? (or this might be github making it look like this?)